packages/console/src/pages/ConnectorDetails/components/ConnectorContent.tsx

```diff
@@ -60,9 +66,11 @@ const ConnectorContent = ({ isDeleted, connectorData, onConnectorUpdated }: Prop
       return;
     }
 
+    const basicBody = { config: result.data, syncProfile: syncProfile === 'EachSignIn' };
+
     const body = connectorData.isStandard
-      ? { config: result.data, metadata: { ...metadata, name: { en: metadata.name } } }
-      : { config: result.data };
+      ? { ...basicBody, metadata: { ...metadata, name: { en: metadata.name } } }
+      : basicBody;
```

Comment:
Nit: maybe introduce some variables to make it more straightforward.
E.g.
```js
const connectorPayload = {.....};
const standardConnectorPayload = {...connectorPayload, {xxxx}};
const body = connectorData.isStandard ? standardConnectorPayload : connectorPayload;
```

Reply:
updated